src/main/java/domain/LotteryGame.java

```diff
@@ -5,46 +5,68 @@
 import java.util.Map;
 import java.util.stream.Collectors;
 
+import domain.factory.LotteryNumberFactory;
 import domain.generatestrategy.LotteryGenerateStrategy;
+import domain.generatestrategy.ManualLotteryGeneratorStrategy;
 
 public class LotteryGame {
 
-	private final PurchaseAmount purchaseAmount;
+	private final PurchaseInformation purchaseInformation;
 	private final LotteryGenerateStrategy lotteryGenerator;
+	private final LotteryNumberFactory lotteryNumberFactory;
+
 	private Lotteries lotteries;
 	private WinningLottery winningLottery;
 
-	public LotteryGame(final PurchaseAmount purchaseAmount, final LotteryGenerateStrategy lotteryGenerator) {
-		this.purchaseAmount = purchaseAmount;
+	public LotteryGame(final PurchaseInformation purchaseInformation, final LotteryGenerateStrategy lotteryGenerator,
+			final LotteryNumberFactory lotteryNumberFactory) {
+		this.purchaseInformation = purchaseInformation;
 		this.lotteryGenerator = lotteryGenerator;
+		this.lotteryNumberFactory = lotteryNumberFactory;
+		createLottery();
+	}
+
+	private void createLottery() {
+		lotteries = new Lotteries();
+		createManualLottery();
 		createAutoLottery();
 	}
 
+	private void createManualLottery() {
+		final List<List<Integer>> rawManualLotteries = purchaseInformation.getManualLotteries();
+		final List<Lottery> manualLottery = new ArrayList<>();
+		for (List<Integer> rawLottery : rawManualLotteries) {
+			manualLottery.add((new ManualLotteryGeneratorStrategy(rawLottery, new LotteryNumberFactory()))
+				.getLottery());
+		}
```

Comment:
```suggestion
		final List<Lottery> manualLottery = new ArrayList<>();
		for (List<Integer> rawLottery : rawManualLotteries) {
		         Lottery lottery = (new ManualLotteryGeneratorStrategy(rawLottery, new LotteryNumberFactory()))
				.getLottery()
			manualLottery.add(lottery);
		}
```
이렇게 표현하는게 좀더 명확할거 같아요. 아니면

```suggestion
		final List<Lottery> manualLottery = rawManualLotteries.stream()
				.map(rawLottery -> new ManualLotteryGeneratorStrategy(rawLottery, new LotteryNumberFactory()).getLottery())
				.collect(Collectors.toList());
```
처럼 할 수도 있겠네요

Reply:
확실히 말해주신 코드가 더 깔끔하네요. 저부분은 자동, 수동 로또 생성 메서드를 통합하면서 없어졌습니다.